### ISSUE TITLE
Display visual map of keyboard shortcuts

### ISSUE THREAD

**Issue by alltheseas** [damus-io/notedeck#531]
Relates to keyboard shortcuts https://github.com/damus-io/notedeck/issues/9

**Comment by alltheseas** [comment]
Example non-map - text shortcut listing

https://support.apple.com/en-us/102650

**Comment by alltheseas** [comment]
https://code.visualstudio.com/assets/docs/getstarted/keybinding/keyboard-shortcuts.gif

**Comment by alltheseas** [comment]
@robagreda what do you think of a visual representation in the spirit of ![image](https://github.com/user-attachments/assets/7ae74ec6-dc3e-4d0a-8e50-bf565b2630cb)

**Comment by alltheseas** [comment]
More examples

https://blog.novacare.no/shortcuts-in-visual-studio-cheatsheet/

![image](https://github.com/user-attachments/assets/fd47c8b5-0832-41a4-89f3-5b53fce44d2e)

![image](https://github.com/user-attachments/assets/f228c833-45ff-4958-b19e-af058cb5d467)

![image](https://github.com/user-attachments/assets/c3646850-cfe3-4648-9650-cbf59fca6a3a)

**Comment by alltheseas** [comment]
CS: Source

https://www.researchgate.net/figure/The-default-keyboard-mapping-for-the-game-Counter-Strike-SourceR-Consistency-between_fig1_260386138

![image](https://github.com/user-attachments/assets/73009620-ec5d-4ce2-968e-fb3f7528b9d4)

**Comment by alltheseas** [comment]
Matrix 
![image](https://github.com/user-attachments/assets/81274f29-debf-4c2e-97ea-f8e7a30d15be)

**Comment by jb55** [comment]
yes but I think we should do these when hovering over anything. much more effective than a reference sheet

<img src="https://uploads.linear.app/654afd0b-7dbd-4dc3-8a99-f389521cd1bb/cbb729aa-4a6f-48bf-b599-fc67e41fea02/e13f623d-30c8-4c32-824e-74719fccca8f?signature=eyJhbGciOiJIUzI1NiIsInR5cCI6IkpXVCJ9.eyJwYXRoIjoiLzY1NGFmZDBiLTdkYmQtNGRjMy04YTk5LWYzODk1MjFjZDFiYi9jYmI3MjlhYS00YTZmLTQ4YmYtYjU5OS1mYzY3ZTQxZmVhMDIvZTEzZjYyM2QtMzBjOC00YzMyLTgyNGUtNzQ3MTlmY2NjYThmIiwiaWF0IjoxNzQwNzYxNjc1LCJleHAiOjMzMzExMzIxNjc1fQ.fOkiXG1kSi6d7tS14ag74rk37mhxLEg_rhIhSHrQNJg " alt="Screenshot 2025-02-28 at 08.54.04.png" width="175" data-linear-height="95" />
